common/timeperiods/timeperiods.go

```diff
@@ -111,7 +111,7 @@ func (t *TimePeriodCalculator) calculatePeriods() {
 		return
 	}
 	iterateDateMate := t.start
-	for !iterateDateMate.Equal(t.end) && !iterateDateMate.After(t.end) {
+	for !iterateDateMate.Equal(t.end) && iterateDateMate.Before(t.end) {
```

Comment:
What does mate refer to?

Reply:
:shrug: I just ran the linter and it decided to fix these for me :smile: Though I can see from git that I wrote that :smile: I can change that if you want?